libvast/test/table_slice.cpp

```diff
@@ -0,0 +1,39 @@
+/******************************************************************************
+ *                    _   _____   __________                                  *
+ *                   | | / / _ | / __/_  __/     Visibility                   *
+ *                   | |/ / __ |_\ \  / /          Across                     *
+ *                   |___/_/ |_/___/ /_/       Space and Time                 *
+ *                                                                            *
+ * This file is part of VAST. It is subject to the license terms in the       *
+ * LICENSE file found in the top-level directory of this distribution and at  *
+ * http://vast.io/license. No part of VAST, including this file, may be       *
+ * copied, modified, propagated, or distributed except according to the terms *
+ * contained in the LICENSE file.                                             *
+ ******************************************************************************/
+
+#define SUITE table_slice
+
+#include "vast/table_slice.hpp"
+
+#include "test.hpp"
+
+#include <caf/test/dsl.hpp>
+
+using namespace vast;
+
+TEST(random integer slices) {
+  record_type layout{
+    {"i", integer_type{}.attributes({{"default", "uniform(100,200)"}})}};
+  layout.name("test");
+  auto slices = unbox(make_random_table_slices(10, 10, layout));
+  CHECK_EQUAL(slices.size(), 10u);
+  CHECK(std::all_of(slices.begin(), slices.end(),
+                    [](auto& slice) { return slice->rows() == 10; }));
+  std::vector<integer> values;
+  for (auto& slice : slices)
+    for (size_t row = 0; row < slice->rows(); ++row)
+      values.emplace_back(get<integer>(slice->at(row, 0)));
+  auto [lowest, highest] = std::minmax_element(values.begin(), values.end());
```

Comment:
Cool use of `minmax_element`, we're increasing coverage of the standard library. 😉